src/systems/eu/implementations/vanilla/packages/eu-utility-text/package.json

```diff
@@ -10,7 +10,7 @@
     "@ecl/generic-utility-text": "^2.0.0-alpha.1"
   },
   "devDependencies": {
-    "@ecl/eu-style-typography-paragraph": "^2.0.0-alpha.1"
+    "@ecl/eu-style-typography": "^2.0.0-alpha.1"
```

Comment:
This dependency is not actually useful

Reply:
it's the case for most of the utilities that were depending on this package: it was only used in the examples and since we don't have examples anymore, we should remove the dependency